internal/CLI.hs

```diff
@@ -22,7 +23,10 @@ lint :: Parser Opts
 lint = Lint <$> pathp
 
 expandPlurals :: Parser Opts
-expandPlurals = pure ExpandPlurals
+expandPlurals = ExpandPlurals <$> minifyp
 
 pathp :: Parser FilePath
 pathp = argument str (metavar "filepath")
+
+minifyp :: Parser JSON.Formatting
```

Comment:
It looks like we have the same function in `cli/CLI`. Is there any way we could share these?

Reply:
There's also some duplicate code across the `Main` modules. Raised this to track it: https://github.com/unsplash/intlc/issues/191